tyk-headless/templates/deployment-pmp.yaml

```diff
@@ -88,37 +90,101 @@ spec:
             value: "4000"
           - name: TYK_PMP_PURGEDELAY
             value: "4"
-          {{- if .Values.mongo.enabled }}
-          - name: TYK_PMP_PUMPS_MONGO_TYPE
+{{ if .Values.backend }}
+{{ if eq .Values.backend "postgres" }}
```

Comment:
@vesko-tyk anyway I can simplify this? 

I tried `{{ if and .Values.backend (eq .Values.backend "postgres") }}` but it seems that the and does not short circuit and the second values causes a crash. I am trying to support older values.yamls

Reply:
In go temple all parts are evaluated. You can create a function or leave it like that

Reply:
I see